wallet/src/test_utils.rs

```diff
@@ -312,43 +312,43 @@ pub fn insert_checkpoint(wallet: &mut Wallet, block: BlockId) {
         .unwrap();
 }
 
-/// Insert transaction
+/// Insert a transaction at the current system time.
```

Comment:
```suggestion
/// Inserts a transaction into the local view, assuming it is currently present in the mempool.
///
/// This can be used, for example, to track a transaction immediately after it is broadcast.
```